lms/static/js/learner_dashboard/views/entitlement_unenroll_view.js

```diff
@@ -0,0 +1,157 @@
+(function(define) {
+    'use strict';
+    define(['backbone',
+        'jquery',
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils'
+    ],
+        function(Backbone, $, gettext, HtmlUtils) {
+            return Backbone.View.extend({
+                el: '#unenroll-entitlement-modal',
+
+                triggerSelector: '.js-action-unenroll-entitlement',
+
+                submitButtonSelector: '#unenroll-entitlement-submit',
+
+                trackInfoSelector: '#entitlement-track-info',
+
+                refundInfoSelector: '#entitlement-refund-info',
+
+                errorInfoSelector: '#unenroll-entitlement-error',
+
+                closeButtonSelector: '#unenroll-entitlement-modal .close-entitlement-modal',
+
+                mainPageSelector: '#dashboard-main',
+
```

Comment:
All these selectors should be passed through as options when the Factory is instantiated. This keeps the classes all in a single file making it easier to maintain consistency across the two.